go/interpreter/lfvm/memory_test.go

```diff
@@ -185,3 +186,97 @@ func TestMemory_expandMemory_expandsMemoryOnlyWhenNeeded(t *testing.T) {
 		})
 	}
 }
+
+func TestMemory_getSlice_ErrorCases(t *testing.T) {
+	c := getEmptyContext()
+	c.gas = 0
+	m := NewMemory()
+	_, err := m.getSlice(0, 1, &c)
+	if !errors.Is(err, errOutOfGas) {
+		t.Errorf("error should be errOutOfGas, instead is: %v", err)
+	}
+	_, err = m.getSlice(math.MaxUint64-31, 32, &c)
+	if !errors.Is(err, errGasUintOverflow) {
+		t.Errorf("error should be errGasUintOverflow, instead is: %v", err)
+	}
+}
+
+func TestMemory_getSlice(t *testing.T) {
```

Comment:
nit: please name the property that is covered by this test, not just the function.